models/_generate_layers.py

```diff
@@ -81,7 +81,8 @@ def setup():
         if param.is_module_init_arg():
           print(param.get_module_param_docstring(indent="    "), file=f)
       print('    """', file=f)
-      print(f"    super().__init__({'**kwargs' if issubclass(cls_base, LayerBase) else ''})", file=f)
+      params = get_super_call_kwargs_repr(layer_class)
```

Comment:
The `params` variable should be renamed (see my earlier comment on this).

Reply:
(But this will be obsolete when you take my other suggestion about the refactor.)